conseil-api/src/main/resources/application.conf

```diff
@@ -0,0 +1,34 @@
+include "metadata.conf"
```

Comment:
Actually, currently we have 3 types of configuration:
- `reference.conf` from the `common`,
- `application.conf` from the `api`/`lorre`,
- `developer.conf` from the "outside".

If you want to use `application.conf` while running either of two mentioned modules, what you need to do is simply add `include application.conf` at the very beginning and then you can overwrite whatever `application.conf` brings (which additionally brings `metadata.conf` so you can overwrite it as well in your `developer.conf` that is the idea behind that. I believe I tested that already but feel free to do it as well, if you have some spare time.

Reply:
It seems to make sense, I'm only worried that we tried that in the past and had issues.

But I might be mistaken, we could've missed the `import`?? 🤔 

Let's try it like this and be hopeful.

Reply:
Ok, I guess we must make sure that the docs clearly explain how to do this, to avoid users will get confused.

The custom application.conf file should start with a strict requirement to include the module configuration with a line like
```
include: required(classpath("application"))
```

This specifies that the application file to include is expected as a classpath resource, and that it's required, so the startup will fail if something is not where it should be.

Reply:
Should the same be needed, when building the docker image, which should provide all needed values in the included application file, bundled via the template generation?
Could you please double-check on that @ajuszczak ?

Reply:
Yes, I can check docker out. I believe that in docker's template I'm already importing `application.conf`.